CoffeeChat/Controller/EditAvailabilityViewController.swift

```diff
@@ -0,0 +1,576 @@
+//
+//  EditAvailabilitiesViewController.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/17/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - Default UICollectionView Header
+private class DefaultHeader: UICollectionReusableView {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+// MARK: - Location UICollectionView Header
+private class LocationHeaderLabel: UICollectionReusableView {
+
+    private let label = UILabel()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        label.font = ._16CircularStdBook
+        addSubview(label)
+        label.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+        }
+    }
+
+    required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+
+    func configure(with text: String) { label.text = text }
+
+}
+
+class EditAvailabilityViewController: UIViewController {
+    
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let contentView = UIView(frame: .zero)
+    private var dayCollectionView: UICollectionView!
+    private let dayLabel = UILabel()
+    private var editScrollView: UIScrollView!
+    private var locationsCollectionView: UICollectionView!
+    private let saveBarButtonItem = UIBarButtonItem()
+    private let scheduleLocationLabel = UILabel()
+    private let scheduleTimeLabel = UILabel()
+    private let scheduleLocationSubLabel = UILabel()
+    private var timeCollectionView: UICollectionView!
+    
+    // MARK: - Time CollectionView Sections
```

Comment:
for now this is okay, but it looks like there's a bunch of repeat code from the `SchedulingTimeViewController`. can we open a git issue to look into modularizing the time & day table as its own class that we can easily reuse?